### ISSUE TITLE
Support line breaks for FEEL statements

### ISSUE THREAD

**Issue by MaxTru** [bpmn-io/bpmn-js-properties-panel#879]
### Is your feature request related to a problem? Please describe.

When writing FEEL expressions, it often is handy to split them across multiple lines.

For example:

```
myVariable = null
or
myVariable > 5
```

rather than

```
myVariable = null or myVariable > 5
```

which is hard to read.

* In particular in the Web Modeler, where the prop panel can not be extended, but this is a different topic.

![Screenshot_20230213_102420](https://user-images.githubusercontent.com/42800119/218419810-5e28395b-b52c-42c7-8e0c-35148535e177.png)

When writing multiline statements, the modeler explicitly persists the lineBreaks. E.g.,:

```
"=myVariable = null &#10;or &#10;myVariable &#62; 5"
```

According to my local testing, this works just fine with the Engine.

However, the Modeler itself will expliticly complain, that this statement is not valid.

![image](https://user-images.githubusercontent.com/42800119/218420337-20ef8a06-b567-4c0b-8c2f-8c6c321c1ca7.png)

### Describe the solution you'd like

- [ ] Explicitly test that multi-line FEEL is accepted by the engine (my fuzzy testing reveiled: yes). Potentially agree this as interface with engine.
- [ ] Adjust linting and code support for FEEL to accept multi-line statements

### Describe alternatives you've considered

- [ ] Keep as is (this would make sense if multi-line is explicitly discouraged

**Comment by nikku** [comment]
Thanks for opening this issue. What you found is a grammar bug.

The same statement works properly, if indented or wrapped in brackets:

```
(
  myVariable = null 
  or
  myVariable > 5
)
```

**Comment by nikku** [comment]
Moving this back to backlog for the time being. I was not able to follow-up.

**Comment by nikku** [comment]
I made significant progress on this issue today but was not able to fully complete it. 

Under the hood this is actually a non-trivial topic. For historical reasons, being tricked by old versions of the DMN spec `lezer-feel` would recognize a list of expressions, rather than a single expression. Newlines would sometimes confuse the parse as it needs to always decide if this is a continuation of an existing expression or the start of a new expression.

The solution we chose is to put the parser in full DMN FEEL 1.5 compat mode, parsing single expressions only (https://github.com/nikku/lezer-feel/commit/63890572d1857c3b24747ef54177ab5a251a553c#diff-3410025680394470cd4d356b9dad20b8a39c090592049243a50866cb6d556677, https://github.com/nikku/lezer-feel/blob/master/CHANGELOG.md#100). This implied downstream changes in all supporting libraries.

What is missing is integration into our editing stack:

* [ ] feel-lint
* [ ] feel-editor
* [ ] bpmn-js-properties-panel
